index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="styles.css">
+    <script src="script.js" defer></script>
+    <title> Welcome! </title>
+  </head>
+  <body>
+    <h1>My Introduction &#128123; </h1>
+    <table style="width:50%">
+      <tr>
+        <th></p><strong>I'm Raynelle </strong> </p>
```

Comment:
I think `<th>` stands for table header. Perhaps the tag it is expecting for the cell is `<td>`?

Reply:
I think there's no <td> required for <th> since its a header.